homeassistant/components/sensor/travisci.py

```diff
@@ -0,0 +1,175 @@
+"""
+This component provides HA sensor support for Travis CI framework.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.travisci/
+"""
+import asyncio
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    ATTR_ATTRIBUTION, CONF_SCAN_INTERVAL, CONF_MONITORED_CONDITIONS,
+    STATE_UNKNOWN)
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['TravisPy==0.3.5']
```

Comment:
This library is not async. Please convert all things to use our sync interface.

Reply:
@pvizeli thanks for your review. 

I've updated the code to sync interface.